cryptofeed/rest/gdax.py

```diff
@@ -35,6 +35,21 @@ def _generate_signature(self, endpoint: str, method: str, body = ''):
             'Content-Type': 'Application/JSON',
         }
 
+    def _pagination(self, endpoint: str, body={}):
```

Comment:
the other part of gdax uses '' as the empty/default body. Why not here?

Reply:
it's '' in one place because it needs to be string, it's None in the other place body is supposed to be dict, so defaulting it to that